rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -128,6 +128,12 @@ class PublisherBase
   size_t
   get_intra_process_subscription_count() const;
 
+  /// Get actual qos settings
+  /** \return The number of subscriptions. */
```

Comment:
Please also put a sentence about why this is needed (w.r.t. `SYSTEM_DEFAULT` and evaluating what the defaults ended up being), and mention that it may be `UNKNOWN` if you sill plan to do that.